Jenkinsfile

```diff
@@ -140,7 +140,7 @@ try {
                 }
               }
 
-              if (['master', 'next-major'].contains(env.BRANCH_NAME)) {
+              if (['master', 'next-major', 'v10'].contains(env.BRANCH_NAME)) {
```

Comment:
Does not look like a common scenario, but always dangerous with things that need to be added multiple places. I get that we could do additional testing on non-published builds, but wouldn't it most often follow each other. Further, could maybe be filtered on generic branch naming conventions, e.g. `['master', 'next-major', 'ojo/']` if not ruining other CI/branching naming conventions.

Reply:
Good point. We should probably extract this list into a shared one.
Generally, we only rigorously test `master` and `next-major`. Releasing from `v10` should probably be considered a hack we will revert once we can merge the branch to `next-major`